src/pages/CatEdit.js

```diff
@@ -2,9 +2,11 @@ import React, { Component } from 'react';
 
  class CatEdit extends Component {
   render() {
-    return <div> 
-        <h3> Cat Edit Page </h3>
-    </div>;
+    return (
+        <div> 
+            <h3> Cat Edit Page </h3>
+        </div>
+    )
```

Comment:
This component looks great! All the components should have clean indentation like this.